### PR TITLE
[HDRP] [Docs] update camera docs and what's new page

### PR DESCRIPTION
**Pull request by pmavridis**
### Purpose of this PR
Update camera docs and what's new page

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Documentation~/whats-new-12.md [hunk 1]
```diff
@@ -106,6 +106,17 @@ The screenshot on the left below do not use additive normal blending, whereas th
 
 ![](Images/HDRPFeatures-SurfGrad.png)
 
+### Physical Camera
+
+HDRP 12.0 includes the following physical camera improvements:
+- Many physical camera properties can now be animated with keyframes using a Unity Timeline.
+- The focus distance of the lens was added in the physical camera properties. For better compatibiity with older HDRP versions, this property is used in DoF computations only if the **Focus Distance Mode** in the [Depth of Field](Post-Processing-Depth-of-Field.md) volume component is set to **Camera**. |
```
Comment (Vic-Cooper) [inline com.unity.render-pipelines.high-definition/Documentation~/whats-new-12.md:113]: Added the **Focus Distance** property to the physical camera properties. To improve compatibility with older HDRP versions, this property is only used in DoF computations if the **Focus Distance Mode** in the [Depth of Field](Post-Processing-Depth-of-Field.md) volume component is set to **Camera**.

File: com.unity.render-pipelines.high-definition/Documentation~/HDRP-Camera.md [hunk 2]
```diff
@@ -55,6 +55,7 @@ The HDRP Camera shares many properties with the Unity's [standard Camera](https:
...
 | **Focal Length**    | Set the distance, in millimeters, between the Camera sensor and the Camera lens. Lower values result in a wider **Field of View**, and vice versa. This property affects [Depth of Field](Post-Processing-Depth-of-Field.md) if you set its **Focus Mode** to **Use Physical Camera**. |
 | **Aperture**        | Use the slider to set the ratio of the f-stop or [f-number](Glossary.md#f-number) aperture. The smaller the value is, the shallower the depth of field is and more light reaches the sensor. This property affects [Depth of Field](Post-Processing-Depth-of-Field.md) if you set its **Focus Mode** to **Use Physical Camera**. This property also affects [Exposure](Override-Exposure.md) if you set its **Mode** to **Use Physical Camera**. |
+| **Focus Distance**     | Set the distance of the focus plane from the Camera. For better compatibiity with older HDRP versions, this property is used in DoF computations only if the **Focus Distance Mode** in the [Depth of Field](Post-Processing-Depth-of-Field.md) volume component is set to **Camera**. |
```
Comment (Vic-Cooper) [inline com.unity.render-pipelines.high-definition/Documentation~/HDRP-Camera.md:58]: Generally, it's best not to mention versions in property docs:

Sets the distance of the focus plane from the Camera. This property is only used in DoF computations if the Focus Distance Mode in the Depth of Field volume component is set to Camera.

File: com.unity.render-pipelines.high-definition/Documentation~/whats-new-12.md [hunk 3]
```diff
@@ -106,6 +106,17 @@ The screenshot on the left below do not use additive normal blending, whereas th
...
+
+### Depth Of Field
+
+The quality of the physically-based Depth Of Field has been improved.
```
Comment (Vic-Cooper) [inline com.unity.render-pipelines.high-definition/Documentation~/whats-new-12.md:117]: Remove passive voice: 

Improved the quality of the physically-based Depth Of Field.

**Review by Vic-Cooper — CHANGES_REQUESTED** [review]
Made a few small suggestions.

File: com.unity.render-pipelines.high-definition/Documentation~/whats-new-12.md (hunk 3 above)
Comment (pmavridis) [reply com.unity.render-pipelines.high-definition/Documentation~/whats-new-12.md]: Hey Vic, I forgot to poke you about this. I have integrated all your suggestions, so I guess this PR is OK now, right?

Comment (Vic-Cooper) [reply com.unity.render-pipelines.high-definition/Documentation~/whats-new-12.md]: No problem! in that case yes and I'll give this the green light. Thanks! :)

**Review by Vic-Cooper — APPROVED** [review]
Requested docs changes made - approved!

**Review by sebastienlagarde — APPROVED** [review]

**Pull request merged**